pod/annotations.go

```diff
@@ -626,3 +626,54 @@ func SidecarAnnotations(sidecarName, suffix string) []string {
 		LegacySidecarAnnotation(sidecarName, suffix),
 	}
 }
+
+type PlatformSidecar struct {
+	Name     string
+	Channel  string
+	ArgsJSON []byte
```

Comment:
Out of curiosity, why did we change `string` to `[]byte`?

Reply:
I chose `[]byte` because it's the type `json.Marshal` / `json.Unmarshal` use. I think it's more cosmetic than anything else, and string would also be fine here.